test/built-ins/AsyncDisposableStack/prototype/disposed/length.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2023 Ron Buckton. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-get-asyncdisposablestack.prototype.disposed
+description: >
+  AsyncDisposableStack.prototype.disposed.length value and descriptor.
+info: |
+  get AsyncDisposableStack.prototype.disposed
+
+  17 ECMAScript Standard Built-in Objects
+includes: [propertyHelper.js]
```

Comment:
Include `features: [explicit-resource-management]`

I think this is why the ESMeta job is complaining about this file.

Reply:
Thanks!